### ISSUE TITLE
Cannot autoplay on the newest google chrome.

### ISSUE THREAD

**Issue by zijian85** [clappr/clappr#1639]
**Browser**: Google Chome 66.0.3359.139  64Bit.

**OS**:  Mac OS 10.10.4

**Clappr Version**: 0.2.83

Hi, today I updated my chrome to the newest. The version is  66.0.3359.139  64Bit.

But cannot autoplay now. I have set the autoplay to true and autoplay on older chrome version.

<img width="425" alt="2018-05-11 12 43 35" src="https://user-images.githubusercontent.com/30042173/39907099-1a6f1518-5519-11e8-8e56-3ab9b36e412a.png">

<img width="829" alt="2018-05-11 12 45 13" src="https://user-images.githubusercontent.com/30042173/39907114-2d3e8282-5519-11e8-8e2d-025daa9ba6c0.png">

**Comment by kslimani** [comment]
@zijian85 i think this is due to new autoplay policy with Chrome version 66.

Also, please note that version 0.2.83 should not be used on any environment ([see releases](https://github.com/clappr/clappr/releases/tag/0.2.83)).

Since Google 66, the autoplay policy seems to be based on a "score" which you can check by typing `chrome://media-engagement` in Chrome adress bar.

You may also get the same issue with Safari version 11.

Please note also on Clappr, the promise exception message has been "silenced" some times ago in  commit 2d2f351.

But since these browsers policy changes, the promise should be used to check if player can autoplay or not. _(for example, using similar mechanisms as [can-autoplay](https://github.com/video-dev/can-autoplay))_.

**Comment by Wiicast** [comment]
When I remove the plugins from my player, it works in Chrome, it seems the plugins is the problem.

**Comment by kslimani** [comment]
Here an exemple to handle autoplay, using can-autoplay :

```html
<html>
  <head>
    <title>My App</title>
    <style>
      .player {
        width: 640px;
        height: 360px;
      }
    </style>
    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/can-autoplay@3.0.0/build/can-autoplay.min.js"></script>
    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/clappr@latest/dist/clappr.min.js"></script>
  </head>
  <body>
    <div class="app">
        <div class="player"></div>
    </div>
    <script>
      var player;
      canAutoplay.video().then(function(o) {
        var cap = o.result === true;
        player = new Clappr.Player({
          source: 'http://clappr.io/highline.mp4',
          parent: '.player',
          height: '100%',
          width: '100%',
          autoPlay: cap,
        });
      });
    </script>
  </body>
</html>
```

**Comment by Wiicast** [comment]
Thank you @kslimani for your reply, I tried this code you rpovided, and in Chrome for Windows 10 it didn't load auto, whereas in Chrome for mac it does?

**Comment by kslimani** [comment]
if by "autoload" you mean autoplay, then for Chrome it depends on the version. The new autoplay policy with Chrome was introduced, if i remember correctly, with version 66.

**Comment by towerz** [comment]
The autoplay policy handling written by @kslimani has already been merged and released on `0.2.97`. It should fix this behavior, or at least prevent state inconsistencies.

I'll close this issue for now. If you need any further assistance, please reopen.
